cfme/configure/access_control/__init__.py

```diff
@@ -932,6 +947,9 @@ def step(self):
         self.prerequisite_view.accordions.accesscontrol.tree.click_path(
             self.obj.appliance.server_region_string(), 'Groups')
 
+    def resetter(self, *args, **kwargs):
```

Comment:
Fix for `group.exists` calls that were never refreshing the screen.

Reply:
Does this somehow get called automatically by virtue of being a CFMENavigateStep?

Reply:
Correct, via `CFMENavigateStep.go()`: https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/implementations/ui.py#L499

Default is `use_resetter=True` via composition of nav_args starting with this dict: 
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/implementations/ui.py#L465